### PR TITLE
ugdb: update to 0.1.10.

### PR DESCRIPTION
**Pull request by reedts**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/ugdb/template [hunk 1]
```diff
@@ -1,14 +1,16 @@
 # Template file for 'ugdb'
 pkgname=ugdb
-version=0.1.9
+version=0.1.10
 revision=1
 build_style=cargo
+depends="gdb"
```
Comment (ericonr) [inline srcpkgs/ugdb/template:6]: Not sure I'm in favor, I usually use `ugdb` with `arm-none-eabi-gdb`.

File: srcpkgs/ugdb/template [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 short_desc="Alternative TUI for gdb"
 maintainer="Érico Nogueira <ericonr@disroot.org>"
 license="MIT"
 homepage="https://github.com/ftilde/ugdb"
 distfiles="${homepage}/archive/${version}.tar.gz"
-checksum=c96925da24fa684f6b22f1fee90a83d34c69be0466ae2fd7fa96cd5af7e903a1
+checksum=ccc41aed24f3ed8fd18decbcbb12ffa78542b5de37692a6dc927c08e4eb234bb
 
 case "$XBPS_TARGET_MACHINE" in
 	ppc*) broken="nix-0.7.0 fails to build";;
```
Comment (ericonr) [inline srcpkgs/ugdb/template:16]: This should truly be removable this time around.

File: srcpkgs/ugdb/template (hunk 1 above)
Comment (reedts) [reply srcpkgs/ugdb/template:6]: Not sure about this one either. Then again, we possible can trust in the user to point `ugdb` to some usable `gdb` binary, right?

File: srcpkgs/ugdb/template (hunk 2 above)
Comment (reedts) [reply srcpkgs/ugdb/template:16]: If I'm not overseeing something [here](https://github.com/nix-rust/nix), ppc is still not officially supported by nix though.

File: srcpkgs/ugdb/template (hunk 1 above)
Comment (ericonr) [reply srcpkgs/ugdb/template:6]: Yep, exactly. I'd prefer to leave without any `depends`

File: srcpkgs/ugdb/template (hunk 2 above)
Comment (ericonr) [reply srcpkgs/ugdb/template:16]: Well, but it builds :p 

0.7.0 was the problematic release that didn't even build.

Comment (reedts) [reply srcpkgs/ugdb/template:16]: Okay, removing it

File: srcpkgs/ugdb/template (hunk 1 above)
Comment (reedts) [reply srcpkgs/ugdb/template:6]: Alright

**Pull request merged**